app-editors/ghostwriter/ghostwriter-1.7.3.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PLOCALES="cs de en es fr it ja pl pt_BR ru zh"
```

Comment:
we do not use l10m thingy anymore

Reply:
What should we use now? I was searching a little bit and found this ( https://www.gentoo.org/support/news-items/2016-06-23-l10n-use_expand.html ) which says that L10N was replacing LINGUAS, but nothing about L10N being deprecated

Reply:
It's a new trend to remove PLOCALES. You can ask Michał Górny to get more info.

Reply:
@reagentoo You are alive :) Do you still want to maintain it? I simply took an eye on it because the current version on portage had an annoying bug. But as you have more experience with it if you have time I suppose that it's better if you maintain it, if not I'm willing to maintain it.

Reply:
`L10N` is commonly used for additional (separate) language packs, it is not intended for filtering of shipped (very small) language files. Use of `LINGUAS` in ebuilds is discouraged and not worth the space constraints, users can use `INSTALL_MASK` to strip away unwanted languages.

https://blogs.gentoo.org/mgorny/2016/05/16/how-linguas-are-thrice-wrong/

Reply:
@stkw0 Please finish current work in accordance with the recommendations.

Reply:
@a17r Can you confirm that `L10N` not needed for this package?

Reply:
I removed the l10n thing

Reply:
@reagentoo No `L10N` was what we discussed. ;) Are you okay with adding @stkw0 as proxied maintainer?

Reply:
@a17r @stkw0 I suggest do not remove `l10n_for_each_locale_do` and l10n and generate **all** locales. Drop `PLOCALES` only.

Reply:
With the current version of the ebuild all translations are being installed (I think lrelease is already called from qmake), why we need to do it manually? I never used l10n so I am probably missing something.

Reply:
@stkw0 ok, if you checked this. @a17r please, add him.
But I'll test later.

Reply:
Yes, translations are installed just fine.